src/createIdentityProvider/identityProviders/createGrant.js

```diff
@@ -156,7 +156,7 @@ export function createGrant(
         if (err.response && err.response.status === 401) {
           // When the grant type is a token, there is no way we can
           // refresh the access token. Therefore we will stop immediately.
-          if (grant_type === 'token') {
+          if (grant_type === 'token' || grant_type === 'testing') {
```

Comment:
Nitpick: update the doc block because now you're also adding the `testing` grant type

Nitpick:


```suggestion
	export const GRANT_TYPE_TOKEN = 'token';
	export const GRANT_TYPE_TESTING = 'testing';
```

```suggestion
	import { GRANT_TYPE_TOKEN, GRANT_TYPE_TESTING } from '../constants';
	
	const no_need_to_refresh_token_grants = [grant_type_token, grant_type_testing]; // lol naming

	if (no_need_to_refresh_token_grants.includes(grant_type)) {
		...
	}
```